<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UWatchSeriesOption</class>
 <widget class="QDialog" name="UWatchSeriesOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Graphs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QListWidget" name="graphsList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Graph's series</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="graphsSeriesList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addSerieButton">
            <property name="text">
             <string>Add new series</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeSerieButton">
            <property name="text">
             <string>Remove selected series</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Series properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="serieColor">
          <item>
           <property name="text">
            <string>red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dark red</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>yellow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>orange</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>green</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dark green</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cyan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dark cyan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dark blue</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>magneta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dark magneta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>gray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dark grey</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>black</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>other</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Line width:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="serieLneWidth">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Line type:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="serieLineType">
          <item>
           <property name="text">
            <string>solid line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dot line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dash line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>dash dot line</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="serieName"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
